.github/workflows/build-pgazure-nightlies.yml

```diff
@@ -25,47 +28,46 @@ jobs:
       fail-fast: false
       matrix:
         platform:
-          # We will remove el/8 support for all apps in the future. For this project packaging image should be updated.But we
-          # don't want to invest.
-          # - el/8
-          # temporarily removed from nightlies since pg15 beta is not being downloaded for ol/7
-          # - ol/7
+          - el/7
+          - debian/buster
+          - debian/bullseye
+          - ubuntu/bionic
           - ubuntu/focal
+          - ubuntu/jammy
 
     steps:
       - name: Checkout repository
         uses: actions/checkout@v3
 
       # This step is to fetch the images unanonymously to have higher bandwidth
       - name: Login to Docker Hub
-        uses: docker/login-action@v1
+        uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USER_NAME }}
           password: ${{ secrets.DOCKERHUB_PASSWORD }}
 
       - name: Clone tools branch
-        run: git clone -b pgaz_test --depth=1  https://github.com/citusdata/tools.git tools
-
-      - name: Clone build branch
-        run: git clone -b "all-pg-azure-storage" --depth=1  https://github.com/citusdata/packaging.git packaging
+        run: git clone -b pg_azure_storage_env --depth=1  https://github.com/citusdata/tools.git tools
 
       - name: Install package dependencies
-        run: sudo apt-get update && sudo apt-get install -y libcurl4
+        run: sudo apt-get update && sudo apt-get install libcurl4-openssl-dev libssl-dev python3-testresources
```

Comment:
I suggest you do not remove `-y` here. It may cause headaches in the future randomly.

Reply:
You're right. Fixed r